### ISSUE TITLE
Clarity required regarding what type of groups

### ISSUE THREAD

**Issue by deepayan21** [MicrosoftDocs/OfficeDocs-SharePoint#1744]
In this page and article, in this line, "If the principal username on the target site is a group, enter TRUE. If it is not a group, enter FALSE. Required.", 
1. Please clarify if the word 'group' means a SharePoint group or an AD group? 
2. Is it possible to map an AD group to a SharePoint group in the target site, i.e., all members of the AD group will be populated in a SharePoint group in the target site?
3. How do I map a SharePoint group to a SharePoint group in SPMT?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d8379014-03ff-3ab5-aa1f-5d9c3a02be68
* Version Independent ID: cfabc646-70ea-6b24-7d16-ea0955479aa5
* Content: [Create a user mapping file for data content migration - Migrate to SharePoint and OneDrive](https://docs.microsoft.com/en-us/sharepointmigration/create-a-user-mapping-file-for-data-content-migration#feedback)
* Content Source: [migration/create-a-user-mapping-file-for-data-content-migration.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/migration/create-a-user-mapping-file-for-data-content-migration.md)
* Service: **sharepoint-online**
* GitHub Login: @JoanneHendrickson
* Microsoft Alias: **jhendr**

**Comment by scanum** [comment]
@deepayan21 Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by deepayan21** [comment]
@scannum any updates?

**Comment by deepayan21** [comment]
@scanum any updates?

**Comment by get-itips** [comment]
Hello @deepayan21 

Made some tests in my lab, and the group honored was the AAD Group, and this was used to fill the modified by attribute of the uploaded file.

 @JoanneHendrickson / @MikePlumleyMSFT Can you kindly confirm this ? I appreciate. Thank you

**Comment by get-itips** [comment]
@JoanneHendrickson @MikePlumleyMSFT can you kindly confirm what I experienced in my tests ? Thank you

**Comment by scanum** [comment]
@JoanneHendrickson @MikePlumleyMSFT can you kindly confirm what Andres experienced in his tests? Thank you

**Comment by JoanneHendrickson** [comment]
Sorry for delay.

The line that describes Column C has been updated to clarify that this is an AD group, not a SharePoint group.  After checking with the product team, they said it is also not possible to map an AD group to a SharePoint group in the target site. Currently it is also not possible to map a SharePoint group to a SharePoint group in SPMT.  

I have updated the topic to reflect this.
